tests/EndToEnd/BasicProvider/TestBasicProvider.cmd

```diff
@@ -0,0 +1,60 @@
+@echo off
```

Comment:
A bit unfortunate to see .CMD files being added.  We should be trending to complete removal of .CMD so we can test across multiple platforms

Reply:
@ Well ... it's little tricky to use the desktop compiler on other platforms.  And these don't quite fit with any of the existing tests because they require both compilers to have been built.

Reply:
It's not a huge issue, we can dig out this debt later.  However we do have a cross-platform desktop compiler that runs on Linux and OSX and Microsoft ship that as one of the dependencies of a Microsoft product (VSforMac). Type providers are used by that compiler in product scenarios, and the code is hosted in VSforMac.  

That scenario should ideally be placed under test here, or we should be on a trajectory to have it under test.